src/main/java/frc/robot/subsystems/DrivetrainSubsystem.java

```diff
@@ -511,6 +511,7 @@ public SwerveModulePosition[] getModulePositions() {
    * @param states array of states. Must be ordered frontLeft, frontRight, backLeft, backRight
    */
   public void setModuleStates(SwerveModuleState[] states) {
+    states[1].speedMetersPerSecond = states[1].speedMetersPerSecond * 13 / 11;
```

Comment:
Please leave a comment here explaining what you are doing.   Otherwise you will be so confused when the other bot drives badly.